ekncontent/ekncontent/eknc-utils-private.h

```diff
@@ -0,0 +1,18 @@
+/* Copyright 2016 Endless Mobile, Inc. */
+
+#pragma once
+
+#include <gio/gio.h>
+#include <json-glib/json-glib.h>
+
+G_BEGIN_DECLS
+
+void
+eknc_utils_append_gparam_from_json_node (JsonNode *node,
```

Comment:
erg do I have to :P
One of my least favorite parts of gnome style because it always ends up breaking with time. https://git.gnome.org/browse/gtk+/tree/gtk/gtkwidget.h#n1110

I had vaguely planned on using a new style here--just copy the decl from your c file, add a semi colon, done.